src/app/app.component.ts

```diff
@@ -52,5 +52,14 @@ export class AppComponent {
 
     console.log(listaAGuardar);
   }
+  duracion(tiempo : number){
+    this.DatosService.tiempodemovimiento += tiempo;
+    if (this.DatosService.tiempodemovimiento >= 61) {
```

Comment:
Como hablamos, yo cambiaría esta linea a 
```suggestion
    if (this.DatosService.tiempodemovimiento > 60) {
```

y la de abajo igual

Reply:
Hecho.